cmd/modern/root.go

```diff
@@ -67,8 +70,8 @@ func (c *Root) SubCommands() []cmdparser.Command {
 		cmdparser.New[*root.Uninstall](dependencies),
 	}
 
-	// BUG:(stuartpa) - Add Mac / Linux support
-	if runtime.GOOS == "windows" {
+	// BUG(stuartpa): - Add Linux support
```

Comment:
>BUG(stuartpa): - Add Linux support [](http://example.com/codeflow?start=4&length=34)

it might be cleaner to just link to an issue and not put an individual alias in the code

Reply:
This is the approach supported by the doc tools

[https://go.dev/blog/godoc](https://go.dev/blog/godoc)

"Top-level comments that begin with the word "BUG(who)” are recognized as known bugs, and included in the “Bugs” section of the package documentation. The “who” part should be the user name of someone who could provide more information.  For example, this is a known issue from the bytes package:"

Agree, the best way to not be contacted is to remove the comment and enter an issue.  But I'd like to encourage this  default approach as the lowest friction way to capture information in the head of folks that have it that might be useful to others.

Reply:
SQL Server source code is full of bug comments attributed to individuals who have long since left the company that seeing names in code just makes me twitch.